community_supplied/Chassis/netconf-system-cpu-temperatures.rule

```diff
@@ -0,0 +1,147 @@
+/*
+ * Detects Chassis components temperatures and notifies when anomalies are found.
+ * One input control detection
+ * 
+ *   1) "threshold" is the threshold that causes the rule to report
+ *      an anomaly.  By default it's 80. This rule will set a dashboard
+ *      color to red when *all* the CPU/Buffer memory/ Heap memory utilization greater than
+ *      'threshold'. 
+ */
+healthbot {
+    topic chassis.temperature  {
+        rule check-netconf-chassis-temperatures {
+	    synopsis "Chassis component temperature change detector";
+            description "Collects temperature details from chassis components and notify anomalies based on threshold values ";
+		    /*
+            * Use componet-name as key for rule.
+            */
+            keys component-name;
+            function temp_degrees {

```

Comment:
Please use - instead of _ for function definitions. 
Can you also rename this as temperature-in-degrees?

Reply:
Modified